### PR TITLE
move to template approach for js

### PR DESCRIPTION
**Pull request by micahgodbolt**

### CODE REVIEW

File: src/view/jsView.js [hunk 1]
```diff
@@ -5,23 +5,29 @@
 
 import { diceValueKey } from "../app";
 
-export const jsRenderView = (diceMap, elem) => {
+const template = document.createElement("template");
+
+template.innerHTML = `
```
Comment (skylerjokiel) [inline src/view/jsView.js:10]: Why not just update the HTML file?

Comment (micahgodbolt) [reply src/view/jsView.js:10]: we've been down that path before, and I think the desire was to keep everything in the JS file and rely less on "magic" in the HTML file.

Also, for React and View examples, the extra DOM in the HTML file won't help.

**Review by skylerjokiel — APPROVED** [review]
My only feedback would be to use id instead of class because `getElementbyId` is cleaner imo.

**Comment by micahgodbolt** [comment]
decided to stick with classes and querySelector for the point that multiple dice might be on the page at the same time. getElementById is document based, querySelector works on a single part of the DOM tree.

**Pull request merged**